.github/workflows/e2e-embed.yml

```diff
@@ -0,0 +1,93 @@
+name: E2E test - embed
+on:
+  push:
+    branches: [ tests/ci-embed ]
+  pull_request_target: # So we can test on forks
+    branches:
+      - main
+    paths-ignore:
+      - public/static/locales/**
```

Comment:
Also make sure to update these paths so this tests only run when embed-related code changes.

Reply:
Thanks for the tip